JSQMessagesViewController/Controllers/JSQMessagesViewController.h

```diff
@@ -268,4 +268,12 @@
  */
 - (void)scrollToBottomAnimated:(BOOL)animated;
 
+/** 
+ * Scrolls the collection view such that the cell in the given row is completely visible, above the `inputToolbar`.
+ *
+ * @param row The index of the row that will be visible
+ * @param animated Pass `YES` if you want to animate scrolling, `NO` if it should be immediate.
+ */
+- (void)scrollToRow:(NSInteger)row animated:(BOOL)animated;
```

Comment:
Changed it. However I'm not sure if an indexPath is the right fit here. It might be more idiomatic and in line with collection views and table views, but `JSQMVC` always contains only one section, doesn't it?